api/auth.go

```diff
@@ -461,6 +461,85 @@ func teamList(w http.ResponseWriter, r *http.Request, t auth.Token) error {
 	return json.NewEncoder(w).Encode(result)
 }
 
+// title: team info
+// path: /teams/{name}
+// method: GET
+// produce: application/json
+// responses:
+//   200: Info team
+//   404: Not found
+//   401: Unauthorized
+func teamInfo(w http.ResponseWriter, r *http.Request, t auth.Token) error {
+	teamName := r.URL.Query().Get(":name")
+	team, err := auth.GetTeam(teamName)
+	if err != nil {
+		return err
+	}
+
+	_, err = t.Permissions()
+	if err != nil {
+		return err
+	}
+
+	apps, err := app.List(&app.Filter{})
+	if err != nil {
+		return err
+	}
+	includedApps := make([]app.App, 0)
+	for _, app := range apps {
+		for _, appTeam := range app.Teams {
+			if appTeam == team.Name {
+				includedApps = append(includedApps, app)
+				break
+			}
+		}
+	}
+
+	pools, err := provision.ListAllPools()
```

Comment:
With this command the Tsuru API create an `pool constraint` with correct and valid context
- `tsuru pool-constraint-set pool1 team team1`

With this command the Tsuru API create an `pool constraint` with fail and invalid context
- `tsuru pool-constraint-set pool1 slash-gnr team1`

I found this case when used `.ListPoolsForTeam` and not get info about pool, I used `tsuru pool-constraint-set pool1 team1` as command, and the Tsuru API created `team1` as context and not value in database.

I think this case a bug, no there context validate.

What do you think about?